assets/src/blocks/Submenu/SubmenuItems.js

```diff
@@ -0,0 +1,32 @@
+export const SubmenuItems = ({ menuItems }) => {
+
+  const renderMenuItems = (items) => {
+    return items.map(({ anchor, text, style, shouldLink, children }) => (
+      <li key={anchor} className={`list-style-${style || 'none'} ${shouldLink ? "list-link" : "list-heading"}`}>
```

Comment:
Some of the examples don't have an anchor (I think ?), so react raises an error 
```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `SubmenuItems`. See https://fb.me/react-warning-keys for more information.
    in li (created by SubmenuItems)
```
Maybe if there's no anchor we can use the array index

Reply:
Good catch :ok_hand:

Reply:
I used the title as an anchor in the example attributes